src/storage/store/column_chunk.cpp

```diff
@@ -473,6 +473,29 @@ class FixedListColumnChunk : public ColumnChunk {
     }
 };
 
+class InternalIDColumnChunk final : public ColumnChunk {
```

Comment:
Should we move the class to .h ?

Reply:
It's actually not exposed and used somewhere, so I think better kept inside cpp file to reduce header file size.